arch/arc/Kconfig

```diff
@@ -253,6 +253,15 @@ config ARC_USE_UNALIGNED_MEM_ACCESS
 	  to support unaligned memory access which is then disabled by default.
 	  Enable unaligned access in hardware and make software to use it.
 
+config ARC_NO_TLS
```

Comment:
The name is misleading - we are not disabling tls, we just don't use it for current task pointer